### PR TITLE
[MRG] add Hashgraph::get_tags_for_sequence

### PR DESCRIPTION
**Pull request by ctb**
Somewhat redundant with `get_tags_and_positions`.

Probably also need to add `Labelhash::get_labels_for_sequence`.

- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make clean diff-cover` If it introduces new functionality in
  `scripts/` is it tested?
- [x] `make format diff_pylint_report cppcheck doc pydocstyle` Is it well
  formatted?
- [x] Did it change the command-line interface? Only backwards-compatible
  additions are allowed without a major version increment. Changing file
  formats also requires a major version number increment.
- [x] For substantial changes or changes to the command-line interface, is it
  documented in `CHANGELOG.md`? See [keepachangelog](http://keepachangelog.com/)
  for more details.
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?
- [x] Do the changes respect streaming IO? (Are they
  tested for streaming IO?)

### CODE REVIEW

**Comment by ctb** [comment]
Ready for review & merge @luizirber @standage @camillescott @betatim

**Comment by ctb** [comment]
Is coverage reporting still borked? Some of those C++ lines are certainly covered by the tests!

File: src/oxli/labelhash.cc [hunk 1]
```diff
@@ -337,6 +337,34 @@ void LabelHash::get_tag_labels(const HashIntoType tag,
...
+
+        if (kmer_tagged) {
+            tags.insert(kmer);
+        }
+    }
+
+    SeenSet::const_iterator si;
+    for (si = tags.begin(); si != tags.end(); ++si) {
+        _get_tag_labels(*si, tag_labels, found_labels);
+    }
```
Comment (luizirber) [inline src/oxli/labelhash.cc]: ☎️ ☎️ ! 2011 called and said that you can replace this block with
```c++
for (auto si: tags) {
    _get_tag_labels(si, tag_labels, found_labels);
}
```

**Review by luizirber — APPROVED** [review]
LGTM

**Pull request merged**